WordPress/src/main/java/org/wordpress/android/ui/mysite/MySiteFragment.kt

```diff
@@ -105,9 +105,14 @@ class MySiteFragment : Fragment(R.layout.my_site_fragment),
             val maxOffset = appBarLayout.totalScrollRange
             val currentOffset = maxOffset + verticalOffset
 
-            updateCollapsibleToolbar(currentOffset)
+            val percentage = if (maxOffset == 0) {
+                updateCollapsibleToolbar(1)
+                resources.getInteger(R.integer.one_hundred)
+            } else {
+                updateCollapsibleToolbar(currentOffset)
+                ((currentOffset.toFloat() / maxOffset.toFloat()) * 100).toInt()
```

Comment:
❓ Would it be better if we add a static variable with the value `100` and use that instead of an integer value in `integers.xml`? Something like this 
```
   val percentage = if (maxOffset == 0) {
                updateCollapsibleToolbar(1)
                MAX_PERCENT
            } else {
                updateCollapsibleToolbar(currentOffset)
                ((currentOffset.toFloat() / maxOffset.toFloat()) * MAX_PERCENT).toInt()
            }
```

Reply:
Thank you for reviewing & approving the PR,
Yeah we can use static value.. Will do the changes and push the updated code.